packages/styles/src/elements/segmented-button.scss

```
@@ -0,0 +1,195 @@
+@use '../functions/tokens';
+@use '../tokens/components';
+@use '../mixins/utilities';
+@use '../core' as post;
+
+tokens.$default-map: components.$post-segmented-button;
+
+$post-segmented-button-max-count: 8;
+
+.segmented-button-container {
+  container-name: post-segmented-container;
+  container-type: inline-size;
+}
+
+legend {
```

This needs to be scoped to the segmented buttons, otherwise it will hide every legend on the page.

```suggestion
.segmented-button > legend {
```